### DISCUSSION TITLE
DBeaver can't show the table with corrrect Queries

### DISCUSSION THREAD

**Discussion by alter29** [risingwavelabs/risingwave#12130] [General]
I wrote a syntax in dbeaver. Ther is no error with the syntax but on the result panel it's only show the column with no rows. How can I fix them Please help me

https://github.com/risingwavelabs/risingwave/assets/144246195/0069e32b-b6c8-4a98-9c76-2e0906815ac1

**Comment by KeXiangWang** [comment]
Could you please paste the query and data in this discussion? Not sure whether it's an issue or just the query doesn't output anything.
Or if its sensitive, can you simplify the query and data. We can try starting with a simple query, first finding a minimal reproduction.

**Reply by alter29** [reply]
Thanks for your respon. Here's the syntax 

SELECT
  order_items.user_id,
  order_items.product_id,
  orders.gender,
  products.category,
  products.brand,
  users.country,
  users.state
FROM
  order_items
INNER JOIN
  orders ON order_items.order_id = orders.order_id
INNER JOIN
  products ON order_items.product_id = products.id
INNER JOIN
  users ON orders.user_id = users.id;

When i run it, no errors detected. But if u could see in the video above, it's just show the columns name, but no rows data

**Reply by KeXiangWang** [reply]
Thanks for the query. Could you please help check whether no rows is expected? I means that after three INNER JOIN, there may be just no results. One feasible way is, you can run the same query with the same data on a standard PostgreSQL database, check what is the output. If PorsgreSQL also output zero rows, it means this result is expected.

**Reply by alter29** [reply]
I did it on postgreSQL but it still show the same result
U can see in this pict...

The problem is, **if I run the same query in Big Query**, it will run normally. **It will show all of data either clomuns or rows** like what I want

![Screenshot (365)](https://github.com/risingwavelabs/risingwave/assets/144246195/6544c866-1885-4b6e-b0e6-00509a8296c6)

**Reply by KeXiangWang** [reply]
Thanks for the effort and experiment. It helps a lot. Based on this result, It seems to be unrelated to Risingwave as it output the same result as PostgreSQL. Would you mind sending me your email address? We can schedule a Zoom meeting and I can try to help you through screen sharing.

**Reply by alter29** [reply]
thanks for your help. 
Btw, do u know about data analytics? or are u a data analyst?

**Reply by KeXiangWang** [reply]
No, sorry, I'm actually a Kernel developer. I can try finding someone more familiar with BigQuery to participate in our discussion. May I ask if you are considering using Risingwave in your production?

**Reply by KeXiangWang** [reply]
Hi @alter29 , have the problem been solved? Do you want to chat about the problem? Or you can join our [community](https://risingwave-community.slack.com/join/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw#/shared-invite/email) and post the questions. More people will notice it and be involved in our discussion.

**Reply by KeXiangWang** [reply]
Hi @alter29 . Since it has been four days with no updates, I'm going to close the discussion. If you have any further questions, please join our [Slack channel](https://risingwave-community.slack.com/join/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw#/shared-invite/email).
